src/libexpr/eval.hh

```diff
@@ -35,6 +37,7 @@ struct PrimOp
     const char * doc = nullptr;
 };
 
+typedef std::map<std::string, Value *> valmap;
```

Comment:
`valmap` won't be scanned for GC roots, is that a problem? If so it should use `std::map<..., traceable_allocator<...>>`.

Also `valmap` should be `ValMap` or something like that to follow our naming conventions.

Reply:
Ok its ValMap now and traceable_allocator